orchestration/dags/dependencies/import_analytics/sql/analytics/enriched_institution_data.sql

```diff
@@ -249,3 +250,5 @@ FROM
         ON rurality.geo_code = eple.code_commune
     LEFT JOIN `{{ bigquery_analytics_dataset }}.institution_locations` as location_info 
         ON educational_institution.institution_id = location_info.institution_id
+    LEFT JOIN `{{ bigquery_analytics_dataset }}.eple_aggregated_type`
```

Comment:
```suggestion
    LEFT JOIN `{{ bigquery_raw_dataset }}.eple_aggregated_type`
```
+ du coup on peut supprimer la query `orchestration/dags/dependencies/cold_data/sql/analytics/eple_aggregated_type.sql`

Reply:
Oui :+1 je pense pas utile d'avoir la table en analytics